python_modules/libraries/dagster-sdf/dagster_sdf/resource.py

```diff
@@ -458,9 +473,10 @@ def cli(
 
         args = [
             self.sdf_executable,
-            *self.global_config_flags,
             *log_level_args,
+            *log_form_args,
             *args,
+            *self.global_config_flags,
```

Comment:
The point of global config flags is to configure the flags passed to `sdf` before a specific command is invoked. So this should come right after the executable.

It should essentially only accept these options:

```
❯ sdf --help          
SDF: A fast SQL compiler, local development framework, and in-memory analytical database

Usage: sdf [OPTIONS] <COMMAND>

Commands:
  new      Create a new sdf workspace
  clean    Remove artifacts that sdf has generated in the past
  compile  Compile models
  run      Run models
  test     Test your models
  stats    Statistics for your data
  report   Report code quality
  check    Check code quality
  lineage  Display lineage for a given table and/or column
  push     Push a local workspace to the SDF Service
  system   System maintenance, install and update
  auth     Authenticate CLI to services like SDF, AWS, OpenAI, etc
  man      Display reference material, like the CLI, dialect specific functions, schemas for authoring and interchange
  dbt      Initialize an sdf workspace from an existing dbt project
  help     Print this message or the help of the given subcommand(s)

Options:
      --log-level <LOG_LEVEL>
          Set log level
          
          [possible values: trace, debug, info, warn, error]

      --log-file <LOG_FILE>
          Creates or replaces the log file

      --log-form <LOG_FORM>
          Set the log format; default flat

          Possible values:
          - flat:   Writes nd-json, compact, no nested objects
          - nested: Writes nd-json, compact, nested objects
          - pretty: Writes nd-json, pretty, nested objects
```

Reply:
wires got crossed here 🤤 you're absolutely right. We should likely have `--show=none` as a global config as well but I'm checking internally why we haven't made that call